tox.ini

```diff
@@ -87,4 +87,23 @@ setenv=
     {[testenv]setenv}
     DJANGO_SETTINGS_MODULE=cfgov.settings.test
 commands=
-    ./manage.py runscript missing_migrations 
+    ./manage.py runscript missing_migrations
+
+[testenv:validate-assets]
+# Invoke with: tox -e validate-assets
+# Ensure all assets are generated without error.
+envdir={[testenv]envdir}-validate-assets
```

Comment:
I would leave this as `envdir={[testenv]envdir}` like the other envs, then tox doesn't have to set up a brand new one just for this. That setup time is part of why it takes so long.

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3537/commits/70d709ce9ce0ce058da1cfa9761914817d8341ca